src/bezpath.rs

```diff
@@ -24,6 +30,15 @@ impl VelloBezPath {
         self.trace = trace;
         self
     }
+
+    /// Gets the progress of/and [`kurbo::PathEl`] which the [`VelloBezPath`] is inbetween at `t`
+    fn inbetween(&self, t: f64) -> (&[PathEl], f64) {
```

Comment:
could return a nested tuple instead of `&[PathEl]` because we _can_ garuntee that there will be two points if we change things only a bit. not sure whats better